<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>feedback</class>
 <widget class="QMainWindow" name="feedback">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>917</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>695</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>1000</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>feedback</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>50</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24px &quot;Sans Serif&quot;;
color: rgb(68, 68, 68);</string>
    </property>
    <property name="text">
     <string>问题反馈</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>125</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>系统问题</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>意见建议</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>商务合作</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>其他</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>165</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px ;
color: rgb(68, 68, 68);
</string>
    </property>
    <property name="text">
     <string>问题描述</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>320</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 255, 250);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>300</width>
       <height>120</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;dfddfd&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;uyghjg&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 244, 244);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>331</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px;
color: rgb(68, 68, 68);
</string>
    </property>
    <property name="text">
     <string>邮箱</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>166</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Sans Serif&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>333</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Sans Serif&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 244);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>440</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px ;
color: rgb(68, 68, 68);
</string>
    </property>
    <property name="text">
     <string>上传附件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244,244,244);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="browse_button" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>440</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px;
background-color: rgb(233, 233, 233);
color: rgb(68, 68, 68);
border:4px ;</string>
    </property>
    <property name="text">
     <string>浏览...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>570</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px;</string>
    </property>
    <property name="text">
     <string>同意获取我的</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>600</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18px;
color: rgb(255, 255, 255);
background-color: rgb(233, 233, 233);
border:4px ;</string>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="systeminfo_button" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>570</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>71</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px;
color: rgb(85, 85, 255);</string>
    </property>
    <property name="text">
     <string>系统信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>125</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px;
color: rgb(68, 68, 68);
</string>
    </property>
    <property name="text">
     <string>问题类型</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>593</y>
      <width>240</width>
      <height>130</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>130</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>130</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-top-left-radius:4px;

border-top-right-radius:4px;

border-bottom-left-radius:4px;

border-bottom-right-radius:4px;
</string>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>220</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>10</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>30</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>50</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>396</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="styleSheet">
        <string notr="true">font: 14px;
spacing: 5px;</string>
       </property>
       <property name="text">
        <string>syslog</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="styleSheet">
        <string notr="true">font: 14px ;
spacing: 5px;</string>
       </property>
       <property name="text">
        <string>apport.log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="styleSheet">
        <string notr="true">font: 14px;
spacing: 5px;
</string>
       </property>
       <property name="text">
        <string>dpkg.log</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>400</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px;
background-color: rgb(255, 255, 255);
color: rgb(68, 68, 68);
</string>
    </property>
    <property name="text">
     <string>日志文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>44</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(136, 136, 136);
font: 12px;</string>
    </property>
    <property name="text">
     <string>限200字</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>文件大小超过了3MB或文件格式不支持</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorMailMessage">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,0,0)</string>
    </property>
    <property name="text">
     <string>请输入正确的邮箱格式</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>systeminfo_button</class>
   <extends>QPushButton</extends>
   <header location="global">systeminfo_button.h</header>
  </customwidget>
  <customwidget>
   <class>browse_button</class>
   <extends>QPushButton</extends>
   <header location="global">browse_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
